### DISCUSSION TITLE
Bare metal install - seems to ignore my config.yaml

### DISCUSSION THREAD

**Discussion by j4nd3r53n** [inventree/InvenTree#4301] [Q&A]
I am trying to set up InvenTree, following the [Bare Metal Setup](https://docs.inventree.org/en/stable/start/install/) document, and it seems to go well, until I reach _Configure InvenTree Options_:

- First, the file _/home/inventree/src/InvenTree/config.yaml_ didn't exist, so I created it and filled it with what I thought was reasonable, based on the [config template](https://github.com/inventree/InvenTree/blob/master/InvenTree/config_template.yaml).
- However, when I run _invoke update_, it breaks, complaining that values I have set in config.yaml aren't set.

I have already found and read [this](https://github.com/inventree/InvenTree/discussions/2897), and followed the instructions to the best of my understanding, but I still have the problem. It isn't just the ENGINE value - it stopped for _INVENTREE_STATIC_ROOT not specified_ as well as for _INVENTREE_BACKUP_DIR_ and _INVENTREE_MEDIA_ROOT_ - and of course _ENGINE_.

**Comment by SchrodingersGat** [comment]
@j4nd3r53n you have to configure the location where your server expects your configuration file to be. 

When the server launches, in the log you should see a clear indication of *where* it is looking for the configuration file. 

I realized that the docs had an error - there was a hard-coded path to the configuration file (as you have used above) which does not necessarily match to where the server is expecting to see this config file. I have just updated the documentation to fix this mistake, should be a bit clearer now.

**Comment by SchrodingersGat** [comment]
https://docs.inventree.org/en/latest/start/config/#configuration-file-location
